syntax = "proto3";
package ProtoBase;

// [START messages]

// internal
message SubscribeReq
{
  int32 type = 1; // 0:删除，1:增加
  repeated int32 msgTypeList = 2;
  repeated string typeNameList = 3;
}

message SubscribeRsp
{
  int32 errorcode = 1;
}

message HeartbeatReq
{
  int32 cpu = 1;	   // 百分比
  int32 memory = 2;	   // 百分比
}

message HeartbeatRsp
{
  int64 serverTime = 1;    // 服务器时间
}

message CommonRsp
{
	int32 code = 1;
	string msg = 2;
}



